applications/FluidDynamicsApplication/custom_strategies/strategies/fractional_step_strategy.h

```diff
@@ -956,6 +959,17 @@ class FractionalStepStrategy : public SolvingStrategy<TSparseSpace, TDenseSpace,
 
         mReformDofSet = rSolverConfig.GetReformDofSet();
 
+        auto& r_process_info = BaseType::GetModelPart().GetProcessInfo();
+        if (r_process_info.Has(FS_PRESSURE_GRADIENT_RELAXATION_FACTOR)) {
+            mPressureGradientRelaxationFactor = r_process_info[FS_PRESSURE_GRADIENT_RELAXATION_FACTOR];
+            KRATOS_INFO(this->Info()) << "Using fractional step strategy with "
```

Comment:
```suggestion
            KRATOS_INFO("FractionalStepStrategy") << "Using fractional step strategy with "
```
We discussed about using `Info` to print the names in another PR. We agreed that this shouldn't be done as info might contain more information than the name.